### ISSUE TITLE
Docker image not provided

### ISSUE THREAD

**Issue by gitirabassi** [vmware-tanzu/octant#136]
From the `examples/in-cluster/kustomization.yaml` there is a reference to `harbor.bryanl.dev/test/octant` which doesn't work, I'm assuming this is an internal/private registry

Can an image be built and tagged on a docker Hub or Quay as part of the release process?

**Comment by GuessWhoSamFoo** [comment]
This is in-progress and shouldn't be difficult to add into goreleaser.

It should be possible to build an image with the current Dockerfile and use it with kustomize as well.

**Comment by gitirabassi** [comment]
can I provide some help?

**Comment by GuessWhoSamFoo** [comment]
@gitirabassi Sure thing! We love to get a PR that builds a docker image in the release process. (And get some feedback/docs about running CI for forks as well). Then I can figure out the remaining piece on which image registry and where credentials should live.

**Comment by zetaab** [comment]
there are some problems with docker:

```
% docker run -it -v ~/.kube/:/kube  octant:latest ./octant --kubeconfig /kube/config
2019-08-14T06:15:32.739Z	INFO	module/manager.go:75	registering action	{"component": "module-manager", "actionPath": "deployment/configuration", "module-name": "overview"}
2019-08-14T06:15:32.742Z	INFO	api/content.go:52	Registering routes for overview
2019-08-14T06:15:32.743Z	INFO	api/content.go:52	Registering routes for cluster-overview
2019-08-14T06:15:32.743Z	INFO	api/content.go:52	Registering routes for configuration
2019-08-14T06:15:32.743Z	INFO	dash/dash.go:326	Dashboard is available at http://127.0.0.1:41319

2019-08-14T06:15:32.744Z	WARN	dash/dash.go:330	unable to open browser: exec: "xdg-open": executable file not found in $PATH
github.com/vmware/octant/internal/dash.(*dash).Run
	/workspace/internal/dash/dash.go:330
github.com/vmware/octant/internal/dash.Run.func1
	/workspace/internal/dash/dash.go:185
```

1) we should be able to disable xdg-open when running inside docker
2) dashboard port is randomly chosen? Its difficult to forward port if its random. I tried to use --ui-url parameter but it does not display anything (perhaps because of xdg-open?)

**Comment by kacole2** [comment]
@zetaab this should be fixed with #199 and using `OCTANT_DISABLE_OPEN_BROWSER=1` when the next version is shipped. It also looks like there is some work with this example. https://github.com/vmware/octant/tree/master/examples/in-cluster

I agree having this run in a docker image and then moving to run in-cluster is a great use-case.

**Comment by wwitzel3** [comment]
Just a heads up, closing this, more details can be found in #450.
